pyproject.toml

```diff
@@ -85,6 +85,7 @@ dependencies = [
   # and bug fixes. We fix the version for improved stability and manually update
   # if necessary.
   "chipwhisperer@https://github.com/newaetech/chipwhisperer-minimal/archive/2643131b71e528791446ee1bab7359120288f4ab.zip",
+  "ruff>=0.9.6",
```

Comment:
This is added to the wrong group

Reply:
Oops https://github.com/lowRISC/opentitan/pull/26349